packages/env/lib/env.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' );
+const dockerCompose = require( 'docker-compose' );
+const NodeGit = require( 'nodegit' );
+const wait = require( 'util' ).promisify( setTimeout );
+
+/**
+ * Internal dependencies
+ */
+const createDockerComposeConfig = require( './create-docker-compose-config' );
+
+// Config Variables
+const pluginPath = process.cwd();
+const pluginName = path.basename( pluginPath );
+const pluginTestsPath = fs.existsSync( './packages' ) ? '/packages' : '';
```

Comment:
Isn't it too much tied to the current structure of Gutenberg repository?

Reply:
yes, I feel some config for these things is necessary:

 - wordpress local install (could be useful for core)
 - test plugins path
 - current plugin path

Reply:
The idea is to detect whether you are running in a plugin, theme, or core and react accordingly.

This part assumes that tests are at the root folder or inside a packages folder.

We'll build better heuristics in follow up PRs.